modules/ROOT/pages/install-checklist.adoc

```diff
@@ -2,108 +2,161 @@
+|TCP | 10249 |Kubernetes kube-proxy |localhost | localhost
+|TCP | 10251-10252 |Kubernetes controller-manager and scheduler |Anypoint Platform PCE nodes | Anypoint Platform PCE nodes
+|TCP | 10257-10259 |Kubernetes controller-manager and scheduler (secure) |Anypoint Platform PCE nodes | Anypoint Platform PCE nodes
+|TCP | 30000-32767 |Internal services port range |Anypoint Platform PCE nodes | Anypoint Platform PCE nodes
+|TCP | 8472 | Overlay VXLAN network |Anypoint Platform PCE nodes | Anypoint Platform PCE nodes
+|TCP | 30080 | HTTP public access |Load balancer  | Anypoint Platform PCE nodes
+|TCP | 30443| HTTP public access |Load balancer  | Anypoint Platform PCE nodes
+|TCP | 30889| Mule Runtime Websocket |Load balancer  | Anypoint Platform PCE nodes
+|TCP | 30883| Mule Runtime Authentication Service (for certificate renewals) |Load balancer  | Anypoint Platform PCE nodes
 |===
 
-== Verify Environment
+For more information, see the provider specific requirements:
+
+* https://docs.rke2.io/install/requirements#operating-systems[Rancher Kubernetes Engine (RKE2) - Networking]
+* https://docs.openshift.com/container-platform/4.16/networking/understanding-networking.html[RedHat OpenShift - Networking]
+* https://anywhere.eks.amazonaws.com/docs/clustermgmt/networking/networking-and-security/[Amazon EKS Anywhere (EKS-A) - Networking]
+
+== NFS Storage Requirements
+
+See xref:verify-nfs.adoc[]
+
+== Container Registry Requirements
+
+These are the requirements:
+
+* Must be secured over TLS.
+* Must be accessible from installer nodes and Kubernetes clusters.
+* Must be secured with either username and password orcCertificate based authentication. Ensure that the new or updated certificates are distributed to all nodes to facilitate uninterrupted TLS communication.
+
+== Installer Node Requirements
+
+These are the requirements:
+
+* Operating system: RHEL, Ubuntu or any Linux based systems.
+* Storage required minimum 100GB.
+* Access to Kubernetes cluster using kubeconfig.yaml
+    ** The cluster needs `cluster:admin` access, as Anypoint Platform PCE installer needs to create resources of different types.
+    ** The cluster needs `read/write` access in the default namespace to run Anypoint Platform PCE installer jobs and other supporting services.
+* Access and connectivity to private container registry
+
+== Storage Requirements
+
+These are the requirements:
+
+In Anypoint Platform PCE version 4.0 and later, all the persistent data is stored in the Persistent Volume Claims (PVCs). Kubernetes Storage Class commands the PVCs. Currently there are two services that require PVCs:
+
+* Object Store - S3, such as object store service
+* Stolon - PostgressSQL database
+
+The storage class for the previous services can be the same. Depending on the type of underlying infrastructure or storage solution you choose, the storage class configuration can vary.
```

Comment:
I do not understand the requirement here. What must the user do/install? Do they need to configure a PVC? Is there any way to clarify this?